sgxs-tools/src/sgx_detect/tests/mod.rs

```diff
@@ -819,6 +919,21 @@ impl Tests {
             self.functions.lookup::<Psw>().supported() == Status::Supported {
             println!("\nYou're all set to start running SGX programs!");
         }
+
+        // Creating two variables as mutex lock could not be taken up twice in a single expression
+        let eos_pre_install_check = {*BUILD_FOR.lock().unwrap() == BuildType::EnclaveOSPreInstall};
+        let eos_post_install_check= {*BUILD_FOR.lock().unwrap() == BuildType::EnclaveOSPostInstall};
```

Comment:
Probably a moot point, but
 - the `{ }` are not needed on the above two lines
 - you don't need to lock `BUILD_FOR` twice -- you can instead just assign a local variable of type `BuildType` to `*BUILD_FOR.lock().unwrap()`.